feature-configs/deploy/nmstate/operator_namespace.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    kubernetes.io/metadata.name: openshift-nmstate
+    name: openshift-nmstate
+  name: openshift-nmstate
+spec:
+  finalizers:
+  - kubernetes
```

Comment:
[nit] my yamllint is complaining that it need `---`  and two spaces here (same for all yamls)

Reply:
Let's make the linter happy :wink: